pkg/validate/validate.go

```diff
@@ -85,14 +88,27 @@ func (v *Validator) LogMessages() {
 	}
 }
 
+func (v *Validator) validateDependencyPath(depPath string) {
+	if depPath == "" {
+		return
+	}
+	matchString := "^[[:alnum:]]+([.][[:alnum:]]+)*(/[[:alnum:]]+)*[/]?$"
+	p := regexp.MustCompile(matchString)
```

Comment:
This should be outside the function. There's no point compiling a regexp every time you use it.

Reply:
Addressed